ddtrace/profiling/collector/_traceback.pyx

```diff
@@ -1,22 +1,45 @@
+from cpython.object cimport PyObject
+
+
+IF PY_MAJOR_VERSION >= 3 and PY_MINOR_VERSION >= 9:
+    cdef extern from "<Python.h>":
+        PyObject* PyFrame_GetCode(PyObject* frame)
+        PyObject* PyFrame_GetBack(PyObject* frame)
+        IF PY_MINOR_VERSION >= 11:
+            PyObject* PyFrame_GetLocals(PyObject* frame)
+            int PyFrame_GetLineNumber(PyObject* frame)
```

Comment:
This isn't building on Python 3.11 properly with compilation errors like:
```
Object of type 'PyObject' has no attribute 'co_varnames'
```
Looks like I need to either import `PyFrameObject` directly or change around these types.